ovos_backend_client/version.py

```diff
@@ -1,7 +1,7 @@
 # The following lines are replaced during the release process.
 # START_VERSION_BLOCK
 VERSION_MAJOR = 0
-VERSION_MINOR = 1
+VERSION_MINOR = 2
 VERSION_BUILD = 1
 VERSION_ALPHA = 3
```

Comment:
can you lower build to 0 and alpha to 1 , that will get automatically published on merge

Reply:
So Minor 2, Build 0, alpha 0?